java/vortex-jni/build.gradle.kts

```diff
@@ -161,19 +225,11 @@ val resourceDir =
         "linux-$osArch"
     }
 
-// Create a release build for every platform we care about.
-// Or we distribute different JARs for each platform...not sure the best approach here.
-// Honestly, fat JAR is probably the move. No one cares about JAR size in Java land, portability
-// is more important.
 val copySharedLibrary by tasks.registering(Copy::class) {
```

Comment:
To be clear, we don't do any cross compilation, right? As written, we'll get a JAR with one native lib, the one for the host's platform. Publishing for multiple platforms is a future us problem, yeah?

Reply:
Yep, this comment is not deleted b/c the problem is solved, it is deleted out of shame